index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="css/normalize.css">
+    <link rel="stylesheet" href="css/styles.css">
+    <link rel="icon" href="images/folder.svg">
+</head>
+<body>
+    <header>
+        <nav class="nav">
+            <img src="images/nav-image.svg" alt="" width="111" height="74">
+            <p class="hamburger">&#9776;</p>
+        </nav>
+    </header>
+    <main>
+        <section class="homepage">
+            <div class="text">
+                <h1 class="heading" id="head">Abdulfatai <br />
+                    Badara
+                </h1>
+                <p>Hello! I am a software developer! I can help you build a product, feature or website. 
+                    Take a look at some of my works. If you like what you see and have a project you need coded, 
+                    don’t hesitate to contact me.
+                </p>
+                <button type="button">Start collaboration</button>
+            </div>
+            <div class="img">
+                <img src="images/home.svg" alt="" width="320" height="160" class="home-img" />
+            </div>
```

Comment:
- Most of the images in this section will serve as decoration. Hence I suggest you to use  background-image instead of an image whenever you want to decorate your page

Reply:
I had to do it like that because the image moves to the right in desktop cersion